hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusDeleteTopicHandler.java

```diff
@@ -47,6 +48,11 @@ public ConsensusDeleteTopicHandler() {
         // Exists for injection
     }
 
+    @Override
+    public void pureChecks(@NonNull TransactionBody txn) {
```

Comment:
Probably should add `throws PreCheckException` on all of these, given the interface declares that, and most (if not all) implementations will likewise require it.

Reply:
Removed in all handlers